### PR TITLE
[docs] Document support for testing in edge legacy

### LINKED ISSUE #4666: [docs] Document how to start Edge Legacy when Edge 79+ is installed
### What is your Test Scenario?
Run tests in Edge Legacy when Edge 79+ is installed.

### What are you suggesting?
Document how to install Edge group policies, enable Edge Legacy and re-run the Edge installer:
[Access Microsoft Edge Legacy after installing the new version of Microsoft Edge](https://docs.microsoft.com/en-us/DeployEdge/microsoft-edge-sysupdate-access-old-edge#how-to-enable-a-side-by-side-experience-with-both-versions-of-microsoft-edge)

Then explain how to switch the `microsoft-edge:` protocol handler to start Edge Legacy:
image
 

### What alternatives have you considered?
Removing Edge 79+

### PR DESCRIPTION
**Pull request by anthophobiac**
Closes #4666

### CODE REVIEW

File: docs/articles/documentation/using-testcafe/common-concepts/browsers/browser-support.md [hunk 1]
```diff
@@ -50,6 +51,8 @@ with the [--list-browsers](../../command-line-interface.md#-b---list-browsers) f
 
 To run tests in a different local browser, specify the path to the browser executable file.
 
+> To test in both Microsoft Edge Chromium and Legacy on the same machine, you can [install them side by side](https://docs.microsoft.com/en-us/DeployEdge/microsoft-edge-sysupdate-access-old-edge#how-to-enable-a-side-by-side-experience-with-both-versions-of-microsoft-edge). Once they are both installed, see the [Microsoft Edge Legacy Support](#edge-legacy) section below for information on how to enable testing in the older version.
```
Comment (dirk-pieterse) [inline docs/articles/documentation/using-testcafe/common-concepts/browsers/browser-support.md]:
```suggestion
> To test in Microsoft Edge Chromium and Legacy on the same machine, [install them side-by-side](https://docs.microsoft.com/en-us/DeployEdge/microsoft-edge-sysupdate-access-old-edge#how-to-enable-a-side-by-side-experience-with-both-versions-of-microsoft-edge). Once they are installed, see the [Microsoft Edge Legacy Support](#edge-legacy) section below for information on how to enable testing in older versions.
```

File: docs/articles/documentation/using-testcafe/common-concepts/browsers/browser-support.md [hunk 2]
```diff
@@ -91,6 +94,14 @@ You can search npm for plugins developed by the community. Their names begin wit
 
 You can also create your own plugin. See [Browser Provider Plugin](../../../extending-testcafe/browser-provider-plugin/README.md) for instructions.
 
+## Microsoft Edge Legacy Support
+
+Tests run in Microsoft Edge Legacy if it is the only Edge version installed on the machine.
```
Comment (dirk-pieterse) [inline docs/articles/documentation/using-testcafe/common-concepts/browsers/browser-support.md]:
```suggestion
Tests are run in Microsoft Edge Legacy if it is the only version of Edge installed on the machine.
```

File: docs/articles/documentation/using-testcafe/common-concepts/browsers/browser-support.md [hunk 3]
```diff
@@ -91,6 +94,14 @@ You can search npm for plugins developed by the community. Their names begin wit
...
+
+In a case where both Chromium-based and Legacy versions are available, you can change the default Edge application in **System Settings** to enable testing in the Legacy version.
```
Comment (dirk-pieterse) [inline docs/articles/documentation/using-testcafe/common-concepts/browsers/browser-support.md]:
```suggestion
If Chromium-based and Legacy versions are available, you can change the default Edge application in **System Settings** to enable testing in the Legacy version.
```

File: docs/articles/documentation/using-testcafe/common-concepts/browsers/browser-support.md [hunk 4]
```diff
@@ -91,6 +94,14 @@ You can search npm for plugins developed by the community. Their names begin wit
...
+
+To achieve this, open **Default apps** in **Windows Settings** and scroll down to select the option **Default Apps by Protocol**. Locate `MICROSOFT-EDGE:` protocol and select Legacy Edge.
```
Comment (dirk-pieterse) [inline docs/articles/documentation/using-testcafe/common-concepts/browsers/browser-support.md]:
```suggestion
To do this, open **Default apps** in **Windows Settings** and scroll down to select the option **Default Apps by Protocol**. Set the `MICROSOFT-EDGE:` protocol to Legacy Edge.
```

File: docs/articles/documentation/using-testcafe/common-concepts/browsers/browser-support.md [hunk 5]
```diff
@@ -91,6 +94,14 @@ You can search npm for plugins developed by the community. Their names begin wit
...
+
 ## Nonconventional Browsers
 
 To use a web browser of a different type, you will need a [browser provider plugin](../../../extending-testcafe/browser-provider-plugin/README.md).
```
Comment (dirk-pieterse) [inline docs/articles/documentation/using-testcafe/common-concepts/browsers/browser-support.md]:
```suggestion
To use a different type of browser, use a [browser provider plugin](../../../extending-testcafe/browser-provider-plugin/README.md).
```

File: docs/articles/documentation/using-testcafe/common-concepts/browsers/browser-support.md [hunk 6]
```diff
@@ -50,6 +51,8 @@ with the [--list-browsers](../../command-line-interface.md#-b---list-browsers) f
 
 To run tests in a different local browser, specify the path to the browser executable file.
 
+> To test in Microsoft Edge Chromium and Legacy on the same machine, [install them side-by-side](https://docs.microsoft.com/en-us/DeployEdge/microsoft-edge-sysupdate-access-old-edge#how-to-enable-a-side-by-side-experience-with-both-versions-of-microsoft-edge). Once they are installed, see the [Microsoft Edge Legacy Support](#edge-legacy) section below for information on how to enable testing in older versions.
```
Comment (VasilyStrelyaev) [inline docs/articles/documentation/using-testcafe/common-concepts/browsers/browser-support.md]: One more `#edge-legacy` link here.

Comment (anthophobiac) [reply docs/articles/documentation/using-testcafe/common-concepts/browsers/browser-support.md]: Fixed

**Pull request merged**